doc/combiner-explainer.md

```diff
@@ -119,37 +119,10 @@ while (q.pop(&f)) {
 }
 ```
 
-`offload` is more than `break`; it does `break` but also causes some
-other thread that is currently waiting on a poll to break out of its
-poll. This is done by setting up a per-polling-island work-queue
-(distributor) wakeup FD. The work-queue is the converse of the combiner; it
-tries to spray events onto as many threads as possible to get as much concurrency as possible.
+`offload` offloads to the work to an `executor` which is an internal pool of threads. 
+`offload_combiner_work_to_some_other_thread` makes one of the threads in the
+executor to pick up and continue the work on the combiner.
 
```

Comment:
Can you give pseudocode for offload and executor just like the old version had it for offload and workqueue?